api/tests/unit/environments/identities/test_identities_feature_states_views.py

```diff
@@ -84,3 +84,32 @@ def test_user_with_update_feature_state_permission_can_update_identity_feature_s
 
     # Then
     assert response.status_code == status.HTTP_201_CREATED
+
+
+def test_user_with_manage_identities_permission_can_retrieve_all_feature_states_for_identity(
```

Comment:
As above, I think it's the VIEW_ENVIRONMENT permission that's doing the work here?